src/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser.php

```diff
@@ -80,6 +86,24 @@ public function resolveAnnotationValue(
         ];
     }
 
+    public function getCommentFromRestOfAnnotation(
+        BetterTokenIterator $tokenIterator,
+        string $annotationContent
+    ): string {
+        // we skip all the remaining tokens from the end of the declaration of values
+        while (
+            preg_match(self::END_OF_VALUE_CHARACTERS_REGEX, $tokenIterator->currentTokenValue())
+        ) {
+            $tokenIterator->next();
+        }
+
+        // the remaining of the annotation content is the comment
+        $comment = substr($annotationContent, $tokenIterator->currentTokenOffset());
+        // we only keep the first line as this will be added as a line comment at the end of the attribute
```

Comment:
This solution is not ideal as some info may be lost but could not think of a better alternative, as adding all lines in a single line seemed wrong and adding several line comments seemed also wrong.